c_wrapper/lua-mongoc-collection.c

```diff
@@ -629,3 +629,150 @@ lua_mongo_collection_destroy(lua_State *L)
     }
     return 1;
 }
+
+
+
+int
+lua_mongo_collection_create_index(lua_State *L)
+{
+    bool throw_error = false;
+    bool created = false;
+
+    collection_t *collection;
+    bson_t keys = BSON_INITIALIZER;
+    bson_t weights = BSON_INITIALIZER;
+    bson_error_t error;
+
+    mongoc_index_opt_t opt;
+    mongoc_index_opt_init(&opt);
+    
+    int absolute_luaBSONObjects_index = 2;
+    int keys_index = 3;
+    int options_index = 4;
+
+    if (!(lua_istable(L, keys_index))) {
+        luaL_error(L, "keys parameter must be a table");
+    }
+
+    if (!(lua_isnil(L, keys_index))){
+        throw_error = !(lua_table_to_bson(L, &keys, keys_index, false, absolute_luaBSONObjects_index, &error ));
+        if(throw_error){
+            goto DONE;
+        }
+    }
+
+    if ( lua_istable(L, options_index) ){
+        lua_getfield(L, options_index, "name"); //String
```

Comment:
There's no need to append `//String` to these lines. This is only because we call 
```
if( lua_isstring(L, -9) ){
  opt.name = lua_tostring(L, -9);
}
```
right after. It self documents itself.